src/modules/vault/layout/sidebar.tsx

```diff
@@ -15,7 +15,7 @@ import { VaultBox, VaultDrawer } from '@/modules/vault/components';
 import { useVaultDrawer } from '@/modules/vault/components/drawer/hook';
 import { useWorkspace } from '@/modules/workspace';
 
-import { useSidebar } from './hook';
+import { useSidebar } from '../../../layouts/dashboard/hook';
```

Comment:
this hook are used only sidebar?
include on hooks of this folder, and use infos by context to control hook

Reply:
It was used called unnecessarily two times in the `header` component. 
I removed it and refactored to avoid dup request. Now it is only called insided the context hook `useVaultDetails`